### DISCUSSION TITLE
How to use Piranha like a data container with a relations between data and how to create a new section?

### DISCUSSION THREAD

**Discussion by biapar** [PiranhaCMS/piranha.core#1932] [Q&A]
How to use Piranha like a data container with a relations between data and how to create a new section to manage ( crud ) data?
I tried to customize the Standard Page but I think that it's not the right solution.
Thanks.

**Comment by tidyui** [comment]
Could you specify a bit more what you’re trying to do?

**Reply by biapar** [reply]
I wish use Piranha as a data management and in/out data by RestApi. The backoffice with roles can access to the panel to review the data. Some data have relation with other data.
